openlibrary/templates/merge_queue/merge_queue.html

```diff
@@ -82,14 +80,15 @@ <h1>$_('Community Edit Requests')</h1>
                 </div>
             </td>
             <td id="reviewer-cell-$(r['id'])">
-              $if r['reviewer'] == username:
+              $# Allow unassigning self if status is "Pending" or "Claimed" (status codes 1 and 4, respectively)
+              $if r['reviewer'] and r['reviewer'] == username and (r['status'] in [1, 4]):
                 $r['reviewer'] <span class="mr-unassign" data-mrid="$r['id']">&times;</span>
               $else:
                 $r['reviewer']
             </td>
             <td>
               $# Status code 1 is "Pending"; 4 is "Claimed"
-              $if r['status'] == 1 or r['status'] == 4:
+              $if r['status'] in [1, 4]:
```

Comment:
Should this just be 1 now? Did 4 get deleted?

Reply:
... but we probably want to hide the "Merge" button if the request is claimed by another person? 
I'm going to make that change.  Otherwise, I don't think that there's anything preventing another person from merging a request that is assigned to someone else.